### PR TITLE
BXMSDOC-3983-master: Update the Interacting with processes and tasks document for PAM 7.4

### PR DESCRIPTION
**Pull request by kaldesai**
[Epic](https://issues.jboss.org/browse/BXMSDOC-3982)
[JIRA](https://issues.jboss.org/browse/BXMSDOC-3983)
[Interacting with processes and tasks RHPAM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-3983-PAM-InteractingFinal/)

Please review and verify the following:
- Section 2.4. Claiming a task
- Section 3.3. Managing tasks using default filter
- Section 4.3. Managing process instances using default filter
- Chapter 6. Viewing and adding comments to a task
- Section 3.4. Defining task variables
- Section 4.4. Defining process instance variables

### CODE REVIEW

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the saved filters icon on the left of the page to expand the *Saved Filters* panel.
+In the *Saved Filters* panel, you can view the saved advanced filters.
+. In the *Saved Filters* panel, set a saved process instance filter as the default filter.
++
+.Default filter selection for Process Instances
+image:processes/process-instance-default-filters.png[Default filter selection for Process Instances]
++
+. Click the *Restore default filters* icon from the Saved Filters panel to remove the default filter.
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc]: not sure this is accurate, to select a new default filter, you just need to click on the start icon of the desired filter.
Restore default filters, means deleted all saved filters and restoring the ones that are shipped by default in the product.

File: doc-content/enterprise-only/processes/interacting-with-processes-viewing-adding-comments-proc.adoc [hunk 2]
```diff
@@ -6,9 +6,12 @@ You can add comments to a task and also view the existing comments of a task in
...
-. In the *Comment* field, enter the comment and click *Add Comment*.
+. On the task page, click the *Work* tab or the *Comments* tab.
+. In the *Comment* field, enter the task related comment and click *Add Comment* icon.
++
+All task related comments are displayed in a tabular form in the *Work* as well as *Comments* tab.
+
 [NOTE]
 ====
-All task comments are displayed in a tabular form in the *Comments* tab.
+To select or clear the *Show task comments at work tab* check box, go to the project’s home page, click the *Settings* icon and select the *Process Administration* option. Only users with the `process-admin` role have access to enable or disable this feature.
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/interacting-with-processes-viewing-adding-comments-proc.adoc]: _go to the project’s home page_, that would Business Central home page, not really related to a project in specific.

Comment (cristianonicolai) [reply doc-content/enterprise-only/processes/interacting-with-processes-viewing-adding-comments-proc.adoc]: `process-admin`, that would be 'admin' role.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-default-filters-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Track* -> *Task Inbox* or go to *Menu* -> *Manage* -> *Tasks*
+. On the *Task Inbox* page or the *Manage Tasks* page, click the saved filters icon on the left of the page to expand the *Saved Filters* panel.
+In the *Saved Filters* panel, you can view the saved advanced filters.
+. In the *Saved Filters* panel, set a saved task filter as the default filter.
++
+.Default filter selection for Tasks or Task Inbox
+image:processes/task-default-filters.png[Default filter selection for Tasks or Task Inbox]
++
+. Click the *Restore default filters* icon from the Saved Filters panel to remove the default filter.
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/interacting-with-processes-tasks-default-filters-proc.adoc]: Same comment as process instances.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
+[id='interacting-with-processes-process-instances-default-filters-proc']
+= Filtering process instances using default filters
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc]: Trying to think a better way to describe the actual feature, maybe something like: Select a default filter for Manage Process Instances, wdyt?

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+You can set a process instance filter as a default filter using the *Saved Filter* option in {CENTRAL}.
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc]: Perhaps add the default filter function after this phrase. Something like: A default filter is the one that will be executed every time the page is open by the user.

Comment (kaldesai) [reply doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc]: Ack

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc (hunk 4 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc]: Ack

File: doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc [hunk 6]
```diff
@@ -1,7 +1,7 @@
-[id='interacting-with-processes-process-instances-special-filter-proc']
-= Defining special filters for process instances
+[id='interacting-with-processes-defining-process-instances-variables-advanced-filters-proc']
+= Defining process instance variables using advanced filters
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc]: Define is not the right term here, I would recommend something like show or view.

File: doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc [hunk 7]
```diff
@@ -1,7 +1,7 @@
...
 
-You can use the *Advanced Filters* option in {CENTRAL} to create special filters for process instances. When you create a special filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
+You can use the *Advanced Filters* option in {CENTRAL} to define process instance variables. When you create a filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc]: Same in here, variables are defined beforehand, this should be more like show or view.

File: doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc (hunk 6 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc]: Done

File: doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc (hunk 7 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc]: Done

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,14 @@
+[id='interacting-with-processes-process-instances-default-filters-proc']
+= Managing process instances using default filter
+
+You can set a process instance filter as a default filter using the *Saved Filter* option in {CENTRAL}. A default filter will be executed every time when the page is open by the user.
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Process Instances*.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc]: s/click/go to/

File: doc-content/enterprise-only/processes/interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc.adoc [hunk 9]
```diff
@@ -1,7 +1,7 @@
-[id='interacting-with-processes-process-instances-special-filter-proc']
-= Defining special filters for process instances
+[id='interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc']
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc.adoc]: I expect this ID isn't used elsewhere, but just to be safe, did you check to make sure? If you find any, they will need to be updated.

File: doc-content/enterprise-only/processes/interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc.adoc [hunk 10]
```diff
@@ -1,7 +1,7 @@
...
+= Viewing process instance variables using advanced filters
 
-You can use the *Advanced Filters* option in {CENTRAL} to create special filters for process instances. When you create a special filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
+You can use the *Advanced Filters* option in {CENTRAL} to view process instance variables. When you create a filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc.adoc]: Question - is "over the column" correct, maybe "for the"?

s/over the column `processId`/for the `processId` column

File: doc-content/enterprise-only/processes/interacting-with-processes-viewing-task-variables-basic-filters-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+= Viewing task variables using basic filters
+
+{CENTRAL} provides basic filters to view task variables in *Manage Tasks* and *Task Inbox*. You can view the task variables of the task as columns using *Show/hide columns*.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Tasks* or  go to *Menu* -> *Track* -> *Task Inbox*.
+. On the *Task Inbox* page, click the filter icon on the left of the page to expand the *Filters* panel
+. In the *Filters* panel, select the *Task Name*.
++
+The filter is applied on the current task list.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-viewing-task-variables-basic-filters-proc.adoc]: Question - is the filter acting on the current UI view or is it being applied to the task list?

s/applied on/applied to

**Comment by cristianonicolai** [comment]
@kaldesai apart from the small fixes pointed out already, it looks good to me, thanks.

**Pull request merged**